lib/auth.js

```diff
@@ -1,25 +1,27 @@
-var debug = require('debug')('SIR:auth');
-var passport = require('passport');
-var GitHubStrategy = require('passport-github2').Strategy;
-var express = require('express');
-var router = express.Router();
-
-var clientId = process.env.CLIENT_ID;
-var clientSecret = process.env.CLIENT_SECRET;
-var nodeEnv = process.env.NODE;
-var baseUrl = nodeEnv === 'production' ?
+import bug from 'debug';
+const debug = bug('SIR:auth');
+import passport from 'passport';
+import githubPassport from 'passport-github2';
+const GitHubStrategy = githubPassport.Strategy;
```

Comment:
Lines 4 & 5 can be simplified as `import { GitHubStrategy } from 'passport-github2'`